beetsplug/fetchart.py

```diff
@@ -909,7 +929,7 @@ def __init__(self):
             'cautious': False,
             'cover_names': ['cover', 'front', 'art', 'album', 'folder'],
             'sources': ['filesystem',
-                        'coverart', 'itunes', 'amazon', 'albumart'],
+                        'coverart', 'itunes', 'amazon', 'vgmdb', 'albumart'],
```

Comment:
I'd argue for not changing the defaults here, since VGMDB seems like a relatively niche service, which many users won't need.